src/clean_env_before_mining.py

```diff
@@ -37,15 +38,14 @@ def clean_environment() -> None:
     # Get the directory of the current script
     script_dir = os.path.dirname(os.path.abspath(__file__))
 
-    # Remove generated data and output from the directories
-    data_directory = os.environ['DATA_DIRECTORY']
+    # Get output directory from the env variables
     output_directory = os.environ['OUTPUT_DIRECTORY']
```

Comment:
I see you have moved the OUTPUT_DIRECTORY to be defined among inputs.
There is a missing input definition now. See files action.yml, README.md, index.js.

Reply:
Code refactoring for clean_env_before_mining.py will be part of other issue.

Reply:
This is resolved and outdated. Comment ready to be closed.